packages/shared/src/components/Feed.tsx

```diff
@@ -436,6 +454,13 @@ export default function Feed<T>({
           onRequestClose={closeSharePost}
         />
       )}
+      <ActivateCompanionModal
```

Comment:
We probably should convert this to a lazyModal as it has multiple invocation.
I think we can then even handle some of the shared login inside the companion activation hook or a new simple hook.

Reply:
This should be done now, let me know what you think